api-docs/list-clients.md

```diff
@@ -25,10 +25,13 @@ Notes:
       "firstName": "Mario",
       "lastName": "Luigi",
       "zip": "84107",
-      "birthday": "1981/01/01",
+      "birthday": "1981-01-01",
```

Comment:
The ISO format that we're using for dates has `-` instead of `/`. See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/backend/apis/utils/transform-utils.js#L1 for how to format the date correctly.